src/js/common/schemaform/review/ReviewArrayField.jsx

```diff
@@ -0,0 +1,229 @@
+import React from 'react';
+import _ from 'lodash/fp';
+import Scroll from 'react-scroll';
+
+import { FormPage } from '../FormPage';
+
+import {
+  getDefaultFormState
+} from 'react-jsonschema-form/lib/utils';
+
+const Element = Scroll.Element;
+const scroller = Scroll.scroller;
+
+/* Growable table (Array) field on the Review page
+ *
+ * The idea here is that, because our pattern for growable tables on the review
+ * page is that each item can be in review or edit mode, we will treat each item
+ * as its own form page and this component will handle the edit/review states and
+ * make sure data is properly updated in the Redux store
+ */
+class ReviewArrayField extends React.Component {
```

Comment:
I wasn't sure exactly what to to call this. It's not exactly the equivalent of the other ArrayField, like the normal/review ObjectField components are.